lms/templates/instructor/instructor_dashboard_2/certificates.html

```diff
@@ -0,0 +1,58 @@
+<%! from django.utils.translation import ugettext as _ %>
```

Comment:
@srpearce Most of the new user-facing messages are in this template.  Anything tagged for translation (prefixed with an underscore) is fair game.